pages/streaks.js

```diff
@@ -24,17 +24,14 @@ const StreaksPage = ({ users }) => {
           font-size: 20px !important;
           line-height: 1.625;
         }
-
         h1 {
           color: var(--colors-orange);
           font-family: var(--fonts-display);
-          font-size: 36px;
-          text-align: center;
```

Comment:
Looks awesome! Let's keep this line in :D

Reply:
Sure! :D

Reply:
I added back the line @sampoder, I removed that one by accident, thanks for noticing!